c++ training/W2S1/E3/Source.cpp

```diff
@@ -0,0 +1,93 @@
+#include <iostream>
+
+//Exercise 3 - Shapes and Areas
+//Create two classes : a Circle class and a Square class.For each class, take in the necessary information to calculate its areaand perimeter.Remember to validate input.
+//
+//Ask the user if they want to create a circle or a square.For their pick, ask the necessary information to create the shape.Then, tell them the areaand the perimeter of their shape.
+//
+//Your class should have the following methods :
+//ExampleShape();
+//ExampleShape(/* Params */);
+//float GetArea();
+//float GetPerimeter();
+
+class Circle {
+private:
+	float m_radius;
+public:
+	Circle() {
+		m_radius = 1.0f;
+	}
+	Circle(float radius) {
+		m_radius = radius;
+	}
+	float GetArea() {
```

Comment:
I do realize that the exercises don't have any information about const, but it does instruct you to use const correctness. This is my mistake and will be soon fixed, but please as soon as you get here tomorrow, let me know and I'll give you a short explanation about const, because we do need to use it. Then, you can come back to this review and apply const correctness :)